pkg/handlers/update.go

```diff
@@ -217,6 +217,22 @@ func (h *Handler) UpdateAdminConsole(w http.ResponseWriter, r *http.Request) {
 		Success: false,
 	}
 
+	isKurl, err := kotsadm.IsKurl()
```

Comment:
hmm, seems like there is another function that doesn't return any errors? https://github.com/replicatedhq/kots/pull/2590/files?w=1#diff-eaeff8d34bc12281033f61dfba86f59d58aa83762ef0af09fb275bd5347fd8bfR11

Reply:
i think error is good here.  if error is ignored, we will end up with a broken kots instance.